app/src/test/java/com/xpeho/xpeapp/domain/AuthenticationManagerTest.kt

```diff
@@ -132,7 +130,7 @@ class AuthenticationManagerTest {
     class LoginTests : BaseTest() {
 
         @Test
-        fun `login with valid credentials sets Authenticated`() = runBlocking {
+        fun `login with valid credentials sets Authenticated`() = runBlocking<Unit> {
```

Comment:
```suggestion
@Test
@DisplayName ("GIVEN we have valid credentials, WHEN we login, THEN we are Authenticated")
        fun validCredentialsMakesAuthenticated() = runBlocking<Unit> {
```

Prefer using camelCase method names and use appropriate JUnit annotation for better readability
https://www.baeldung.com/junit-custom-display-name-generator